.github/workflows/twistlock_cronjob.yml

```diff
@@ -0,0 +1,57 @@
+name: Twistlock Cronjob
+
+on:
+  schedule:
+    - cron: '0 10 * * *' # Runs at 5:00 AM EST every day
+  workflow_dispatch:
+
+jobs:
+  twistlock-scan:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.ref }}
+
+      - name: Configure VAEC AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1.7.0
+        with:
+          aws-access-key-id: ${{ secrets.VAEC_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.VAEC_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-gov-west-1
+          role-to-assume: ${{ secrets.VAEC_DEPLOY_ROLE }}
+          role-skip-session-tagging: true
+          role-duration-seconds: 900
+
+      - name: Query ECR for images
+        id: get-image-tags
+        run: |
+          DATE=$(date --date="1 day ago" "+%Y-%m-%d")
+          IMAGES=$(aws ecr describe-images --repository-name notification_api --query "imageDetails[?starts_with(imagePushedAt, '$DATE')].imageTags" --output text)
```

Comment:
We don't want to run those that are "\<untagged\>". Should be able to either filter it here or in the for loop starting on line 48.
![image](https://user-images.githubusercontent.com/16893311/219800737-a24f2b96-b9cc-49b3-adb4-c7909bfe87e4.png)